scss/modules/_nav.scss

```diff
@@ -297,92 +241,134 @@
+// Switches to traditional dropdown menu
+// Hide the submenu toggles
 
-  @include media($lg) {
-    height: 6rem;
+@include media($lg) {
+  .site-nav {
+    padding-left: 2rem;
   }
-}
 
-// Toggle controls for the mobile menu
-// 
-// We use the same logic to show the entire menu as well as the sub-menus (site-nav__dropdown)
-// When on mobile, we hide the site-nav__link in the site-nav__item--with-dropdown elements so that tapping on the toggle opens the dropdown, instead of taking you to the page
-//
-// Elements:
-// .nav-toggle__input                       - The checkbox that controls whether adjacent lists are shown or hidden
-// .nav-toggle__label                       - The label attached to the checkbox (not applied to the main menu button)
-// .site-nav__item--with-dropdown           - The outer li that contains the dropdown list and toggle
+  .site-nav__list {
+    @include clearfix();
+    @include u-bg--neutral();
+    @include transform(translateX(0));
+    background: none;
+    border: none;
+    min-height: 0;
+    display: block;
+    float: left;
+    padding: 0;
+    position: relative;
+    top: 0;
 
-.site-nav__item--with-dropdown {
-  & > a.site-nav__link {
-    display: none;
+    .utility-nav__item {
+      display: none;
+    }
   }
 
-  @include media($lg) {
-    & > a.site-nav__link {
-      display: block;
+  .site-nav__item {
+    float: left;
+    border-bottom: none;
+
+    &.is-open,
+    &:hover {
+      .site-nav__link {
+        border-bottom-color: $primary-contrast;
+        background-color: $primary;
+        color: $ivory;
+      }
     }
 
-    & > .nav-toggle__input,
-    & > .nav-toggle__label {
-      display: none;
+    &.is-disabled {
+      .site-nav__link {
+        color: #737373;
+        cursor: default;
+
+        &:hover,
+        &:focus {
+          background-color: transparent;
+        }
+      }
+
+      &:hover,
+      &:focus {
+        background-color: transparent;
+      }
     }
   }
-}
 
-.nav-toggle__label {
-  position: relative;
-  font-family: $serif;
+  .site-nav__link {
+    padding: 2rem;
 
-  &::after {
-    @include u-background-image('i-right--neutral');
-    content: '';
-    display: block;
-    width: 2rem;
-    height: 2rem;
-    position: absolute;
-    left: -1rem;
-    top: 1rem;
+    &:hover,
+    &:focus,
+    &:active {
+      border-bottom-color: $primary-contrast;
+      background-color: $primary;
+      color: $ivory;
+    }
   }
-}
 
-.nav-toggle__input {
-  @include u-visually-hidden();
+  .site-nav__dropdown {
+    @include u-bg--primary();
+    border: 1px solid $primary-contrast;
+    border-top: none;
+    padding-bottom: 0;
+    position: absolute;
+    top: 6rem;
+    z-index: $z-navigation;
 
-  & + label {
-    margin-bottom: 0;
+    .site-nav__item {
+      width: 100%;
+    }
 
-    &::before {
-      display: none;
+    .site-nav__subtitle {
+      border-top: 1px solid $primary-contrast;
+      padding: 2rem 2rem 0 2rem;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `2rem 2rem 0` instead of `2rem 2rem 0 2rem`